stateproof/README.md

```diff
@@ -0,0 +1,39 @@
+# StateProof
+
+The stateproof package implements the orchestration logic for state proof. 
```

Comment:
actually maybe we can link to the crypto package? (since it's not going to be where the README is)

Reply:
I would really like to start the paragraph by talking about `stateproof` and not `crypto.stateproof`, since `stateproof` is the subject here. I'm okay with rephrasing the sentence if you have an idea.

Reply:
having the background can cover for it.